### PR TITLE
Delete ionia gift cards

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204583307528855

### CODE REVIEW

File: src/constants/WalletAndCurrencyConstants.ts [hunk 1]
```diff
@@ -113,7 +113,6 @@ interface SpecialCurrencyInfo {
 
   // Marketing:
   displayBuyCrypto?: boolean
-  displayIoniaRewards?: boolean
```
Comment (samholmes) [inline src/constants/WalletAndCurrencyConstants.ts:116]: Where was this used?

**Review by samholmes — APPROVED** [review]
I don't necessarily see how this will impact the work over https://github.com/EdgeApp/edge-react-gui/pull/4191 so I'll approve this. I'm unsure if any of this will be leveraged for the new plugin.

File: src/components/themed/ControlPanel.tsx [hunk 2]
```diff
@@ -267,17 +263,6 @@ export function ControlPanel(props: DrawerContentComponentProps) {
...
   ]
 
-  if (!hideIoniaRewards && IONIA_SUPPORTED_FIATS.includes(defaultFiat)) {
-    rowDatas.unshift({
-      pressHandler: () => {
-        navigation.navigate('pluginViewSell', { plugin: guiPlugins.ionia })
-        navigation.dispatch(DrawerActions.closeDrawer())
-      },
-      iconNameFontAwesome: 'hand-holding-usd',
-      title: sprintf(lstrings.side_menu_rewards_button_1s, defaultFiat)
```
[the remark this answers is not in the document]
Comment (swansontec) [reply src/components/themed/ControlPanel.tsx:277]: Fixed.

File: src/constants/WalletAndCurrencyConstants.ts (hunk 1 above)
Comment (swansontec) [reply src/constants/WalletAndCurrencyConstants.ts:116]: EarnCryptoCard.tsx

**Pull request merged**